sdcm/cluster.py

```diff
@@ -2872,16 +2872,16 @@ def _create_session(self, node, keyspace, user, password, compression,
             protocol_version = 3
 
         authenticator = self.params.get('authenticator')
-        if user is None and password is None and (authenticator and authenticator == 'PasswordAuthenticator'):
-            user = self.params.get('authenticator_user', default='cassandra')
-            password = self.params.get('authenticator_password', default='cassandra')
+        if authenticator and authenticator == 'PasswordAuthenticator':
+            user = self.params.get('authenticator_user')
+            password = self.params.get('authenticator_password')
```

Comment:
set the default in `test_default.yaml`

Reply:
Do we want to use credentials in every stress command?